setup-scripts/support/authorized_keys

```diff
@@ -8,3 +8,4 @@ ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIPM43lzbKjFLChe5rKETxDpWpNlqXCGTBPiWlDN2vlLD
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIN6Kcx8yetova4/ALUQHigo/PBMJO33ZTKOsg2jxSO2a user@deploy.siasky.dev
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDcenWnMQ6q/OEC4ZmQgjLDV2obWlR3fENV0zRGFvJF+ marcins@siasky.net
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIB7prtVOTwtcSN9HkXum107RwcW5H8Vggx6Qv7T57ItT daniel@siasky.net
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCyA5V8uLIbp8KGbHOk2EeYlIWnVbw+PAhc1NWvFgPvRk1vY4h6KpPdKdfmtNoYshoQYykLRAZuvQV+n8EE4u9PSFIaOTGtBFJdVBzwC8jAfE2fK+a98JFO3AYZNlGGEyV0pRwkVnbAq7KPo4dcsp0XsWax8k9jpzPlpsUNS8lcjmxddj14BGL3ikPmFSsK5bYjPtJlJvOGxbaoGUp0BxVTs9YkJHhXG2NYQoAhClRR4rdyGBq1S2pyPmY6chUK19FTHbAsxIBWUi2GUfk4OnHyv0TCeZnsqpFisYQzHzCd1xq96dceoDKzWlDgzfvLqDMk5yxpLoe057qufccSy0Ds4BSh304v3atf7P1P6IjsHw2UJwZ78h9VFmr25lDOFN6ITvStZ+v1BqkLzqyfMsp/CM3Jeg91PeXhR9ODUfkD0dB8pH6YXfIkHBQc4XAllYYiPmngL1RnFybU8IfEomkjkGkZnFmB9/mKRjxB5k+OGpehmafDcJJ6LxDWF8X76jtrSODkSnbck5jENEjsUxXKqX/hSU2lwK17632hi7fUwwA5s9Og/7vKHKA5ovGNBo0+gPqqFEPZd9bWLfoyStyu+zNfhhES8mGFOW4/bdshzGUB1ZpXqUiW/MFd5wRTrmPFX8lfX6FkIpRU1EGeCziAi18FHbXKGx3JRo/mVf2kJQ== michal.leszczyk@skynetlabs.com
```

Comment:
Maybe I should consider using `ed25519` 🤔

Reply:
yeah you should

Reply:
Alrighty then ;)